lib/src/utils.dart

```diff
@@ -286,6 +289,33 @@ class Utils {
     return result;
   }
 
+  @internal
+  static FutureOr<String> getNetworkType() async {
+    if (lkPlatformIsTest()) {
+      return 'wifi';
+    }
+    var connectivityResult = await (Connectivity().checkConnectivity());
+    String networkType = 'unknown';
+    switch (connectivityResult) {
+      case ConnectivityResult.mobile:
+        networkType = 'mobile';
+        break;
+      case ConnectivityResult.wifi:
+        networkType = 'wifi';
+        break;
+      case ConnectivityResult.bluetooth:
+        networkType = 'bluetooth';
+        break;
+      case ConnectivityResult.ethernet:
+        networkType = 'ethernet';
+        break;
+      case ConnectivityResult.none:
+        networkType = 'none';
+        break;
+    }
```

Comment:
I think we should follow the spec in the proto?
// wifi, wired, cellular, vpn, empty if not known

Reply:
sure,  I can change it to be consistent
mobile -> cellular
ethernet -> wired
none -> empty

But it seems https://pub.dev/packages/connectivity_plus plugin can't detect VPN, and has a `bluetooth` type

Reply:
yes I can't detect VPN also... I'm just sending empty for now.